templates/centos7/docker.repo.j2

```diff
@@ -0,0 +1,6 @@
+[dockerrepo]
```

Comment:
If disabled by default, you need to specify the name and the README tells its `docker`.
I think it's better to use `docker` to be consistent with the other available repos.